stl/src/tzdb.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// This must be as small as possible, because its contents are
+// injected into the msvcprt.lib and msvcprtd.lib import libraries.
+// Do not include or define anything else here.
+// In particular, basic_string must not be included here.
+
+#include <xtzdb.h>
+
+#include <Windows.h>
+
+#pragma comment(lib, "Advapi32")

```

Comment:
Is this necessary given that you're explicitly linking `advapi32.lib` in the `CMakeLists.txt`?

Reply:
Yes, user code needs to link against `advapi32.lib`, not just the DLL. I'll double-check, but I'm pretty sure the tests failed to build without this.

Reply:
Confirmed, some configurations of the tests won't build without this.